modules/optimization/src/functions/ParsedOptimizationFunction.C

```diff
@@ -22,42 +22,42 @@ ParsedOptimizationFunction::validParams()
       "Function used for optimization that uses a parsed expression with parameter dependence.");
 
   params.addRequiredCustomTypeParam<std::string>(
-      "value", "FunctionExpression", "The user defined function.");
+      "expression", "FunctionExpression", "The user defined function.");
   params.addRequiredParam<std::vector<std::string>>(
-      "param_vars", "Names of parameters in 'value' being optimized.");
+      "param_names", "Names of parameters in 'expression' being optimized.");
   params.addRequiredParam<ReporterName>(
       "param_vector_name", "Reporter or VectorPostprocessor vector containing parameter values.");
   params.addParam<std::vector<std::string>>(
-      "constant_vars",
+      "symbol_names",
```

Comment:
Maybe `constant_symbol_names`? This isn't quite identical to `ParsedFunction` since it only accepts `Real` values at input.

Reply:
constant_names then?

Reply:
I think `constant_symbol_names` works for me.